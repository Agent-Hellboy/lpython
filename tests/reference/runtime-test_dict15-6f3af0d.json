{
    "basename": "runtime-test_dict15-6f3af0d",
    "cmd": "lpython {infile}",
    "infile": "tests/errors/test_dict15.py",
    "infile_hash": "6a0e507b9a9cf659cb433abbdc3435b4c63a6079eadcd7d2c765def1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "runtime-test_dict15-6f3af0d.stderr",
    "stderr_hash": "7d53798a879549d9446d5948c0f4de57e2768f7107cf238c25fa579e",
    "returncode": 1
}