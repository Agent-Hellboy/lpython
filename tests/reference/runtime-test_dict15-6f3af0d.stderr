ld: warning: object file (test_dict15.out.tmp.o) was built for newer macOS version (14.0) than being linked (13.3)
KeyError: The dict does not contain the specified key
